satpy/multiscene.py

```diff
@@ -174,18 +174,29 @@ def first_scene(self):
         return self._scene_gen.first
 
     @classmethod
-    def from_files(cls, files_to_sort, reader=None, **kwargs):
+    def from_files(cls, files_to_sort, reader=None,
+                   ensure_all_readers=False, **kwargs):
         """Create multiple Scene objects from multiple files.
 
+        Args:
+            files_to_sort (Collection[str]): files to read
+            reader (str or Collection[str]): reader or readers to use
+            ensure_all_readers (bool): If True, limit to scenes where all
+                readers have at least one file.  If False (default), include
+                all scenes where at least one reader has at least one file.
+
         This uses the :func:`satpy.readers.group_files` function to group
-        files. See this function for more details on possible keyword
-        arguments.
+        files. See this function for more details on additional possible
+        keyword arguments.  In particular, it is strongly recommended to pass
```

Comment:
I would say `possible additional keyword arguments` but I'm not a native speaker. @djhoese @simonrp84 what would you say?

Reply:
:thinking: Bottom line: It doesn't really matter.

That said, I think what he has is good. I think having `additional` first makes it the more important part of the phrase whereas `possible` being first makes "possible" the more important part. I think "additional" is what we want to emphasize. Again, it doesn't really matter in my opinion.

Reply:
Ok, sorry for the noise then! :)